Kickstarter.xcodeproj/project.pbxproj

```diff
@@ -214,6 +214,12 @@
 		9DEE3B561D1D819D0020C2BE /* ProjectActivityBackingCellViewModel.swift in Sources */ = {isa = PBXBuildFile; fileRef = 9DEE3B241D1D81950020C2BE /* ProjectActivityBackingCellViewModel.swift */; };
 		9DEE3B591D1D81C80020C2BE /* ProjectActivityBackingCellViewModelTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 9DEE3B581D1D81C80020C2BE /* ProjectActivityBackingCellViewModelTests.swift */; };
 		A70119451CD92261009F8F65 /* CommentsDataSourceTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = A70119171CD921D0009F8F65 /* CommentsDataSourceTests.swift */; };
+		A7066A851E27B67000E8658C /* ReactiveSwift.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = A7066A841E27B67000E8658C /* ReactiveSwift.framework */; };
+		A7066A861E27B67000E8658C /* ReactiveSwift.framework in Embed Frameworks */ = {isa = PBXBuildFile; fileRef = A7066A841E27B67000E8658C /* ReactiveSwift.framework */; settings = {ATTRIBUTES = (CodeSignOnCopy, RemoveHeadersOnCopy, ); }; };
+		A7066A881E27B68300E8658C /* Result.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = A7066A871E27B68200E8658C /* Result.framework */; };
+		A7066A891E27B68300E8658C /* Result.framework in Embed Frameworks */ = {isa = PBXBuildFile; fileRef = A7066A871E27B68200E8658C /* Result.framework */; settings = {ATTRIBUTES = (CodeSignOnCopy, RemoveHeadersOnCopy, ); }; };
+		A7066A8B1E27B69700E8658C /* Runes.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = A7066A8A1E27B69700E8658C /* Runes.framework */; };
+		A7066A8C1E27B69700E8658C /* Runes.framework in Embed Frameworks */ = {isa = PBXBuildFile; fileRef = A7066A8A1E27B69700E8658C /* Runes.framework */; settings = {ATTRIBUTES = (CodeSignOnCopy, RemoveHeadersOnCopy, ); }; };
```

Comment:
cool, @cdolm92 and i were waiting to see if you were cool with the automatic signing and embedding these in the main project to get everything running ok.